foundry.toml

```diff
@@ -1,19 +1,9 @@
 [profile.default]
 src = 'contracts'                                             # the source directory
-test = 'test'                                                 # the test directory
-out = 'artifacts'                                             # the output directory (for artifacts)
-cache = true                                                  # whether to cache builds or not
-force = false                                                 # whether to ignore the cache (clean build)
```

Comment:
> Also, remove default configs in file to make it clear.

In my opinion removing default values would not make it clear as we can not memorize default values in our head. At least for the config values we care, we can leave them in the config file so we don't have to run `forge config` every time we want to know what the default value is.

Reply:
Yeah, I agree some default value we care should be listed here. Perhaps only `cache` and `force`?

Reply:
Makes sense!

Reply:
Adjusted 788db72